hedera-node/src/main/java/com/hedera/services/state/merkle/MerkleAccount.java

```diff
@@ -78,6 +79,7 @@ public void setKey(EntityNum phi) {
 		private static final int RELEASE_090_RECORDS = 1;
 		private static final int RELEASE_090_ASSOCIATED_TOKENS = 2;
 		static final int NUM_090_CHILDREN = 3;
+		static final int NUM_0240_CHILDREN = 2;
```

Comment:
We should update `getMinimumChildCount(final int version)` to return `NUM_0240_CHILDREN` appropriately.